runtime/doc/options.txt

```diff
@@ -4588,9 +4588,11 @@ A jump table for the options with a short description can be found at |Q_op|.
 			larger scroll when drawing the message area (with
 			'display' msgsep flag active).
 	    invalid	Enable stricter checking (abort) of inconsistencies
-			of the internal screen state. This is mosly
+			of the internal screen state. This is mostly
 			useful when running nvim inside a debugger (and
 			the test suite).
+	    nodelta	Trigger a delay with all redrawn characters, even if
```

Comment:
Again, this has nothing to do with delays. More something like "Send all internally redrawn cells to UI, even if they are unchanged from the previously displayed state".